cinema/views.py

```diff
@@ -127,6 +125,25 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
This is a code style issue. It's a good practice to handle the negative case first in an if-else statement. So, you should check if serializer.is_valid() is False first, then return the error response. This way, the main logic of your code (i.e., when the data is valid) doesn't get indented, and it improves the readability of your code.